scripts/mergeswag.sh

```diff
@@ -23,6 +27,8 @@ metadata='{
   }
 }'
 
-find "$folder/" -name '*.swagger.json' -print0 \
+find "$src" -type f -name "*" -print0 \
 	| sort -zr \
 	| xargs -0 jq -s 'reduce .[] as $item ('"$metadata"'; $item * .)'
+
+rm -rf $src
```

Comment:
It would be good to create a `trap` for this after `src=$(mktemp -d)`. We should clean up the temp dir even if the script exits earlier.